ui/main/src/app/modules/share/archives-logging-filters/archives-logging-filters.component.html

```diff
@@ -149,7 +149,7 @@
 
             <div style="margin-left: 40px; margin-right: 40px">
                 <div
-                    class="opfab-input"
+                    class="opfab-date-range-input opfab-input"
```

Comment:
![image](https://github.com/user-attachments/assets/01bec395-2b34-4d5e-be1a-92b4b8fddc0b)
![image](https://github.com/user-attachments/assets/6ffebc9c-88a5-45ef-a176-4f9453995eb6)

"publish date range" is not aligned the same way (in relation to the line of the input) than the others ("process" for example). 
The same for "active date range".